source/helpers/math.ts

```diff
@@ -0,0 +1,36 @@
+export function lerp(min: number, max: number, t: number): number {
+	return min + ((max - min) * t);
+}
+
+export function invlerp(min: number, max: number, x: number): number {
+	return (x - min) / (max - min);
+}
+
+export function calculateMinMax(values: number[]): {min: number; max: number} {
+	let min = values[0];
+	let max = values[0];
+	for (let i = 1; i < values.length; i++) {
+		if (values[i] < min) {
+			min = values[i];
+		} else if (values[i] > max) {
+			max = values[i];
+		}
+	}
+
+	return {min, max};
```

Comment:
```suggestion
	return {
		min: Math.min(...values),
		max: Math.max(...values),
	};
```

Reply:
That method is pretty but sucks for medium/large arrays. GH has a lot of large lists so it's better to use the efficient one if this is gonna be a general util. My method is also 1 pass instead of 2.
https://medium.com/coding-at-dawn/the-fastest-way-to-find-minimum-and-maximum-values-in-an-array-in-javascript-2511115f8621

Reply:
There are no such list of any size on GitHub close to making this "inefficient". With numbers always less than 100, we prefer readability over microoptimizations

Reply:
I was thinking of line numbers in files and diffs. But sure, will change.